daemon/src/wallet.rs

```diff
@@ -186,10 +186,15 @@ impl Actor {
     pub fn handle_try_broadcast_transaction(
         &mut self,
         msg: TryBroadcastTransaction,
+        ctx: &mut xtra::Context<Self>,
     ) -> Result<Txid> {
         let tx = msg.tx;
         let txid = tx.txid();
 
+        let this = ctx.address().expect("self to be alive");
+
+        this.send(Sync).await??;
```

Comment:
How is this going to make a difference? Syncing will scan the blockchain for coins sent to addresses derived from the descriptor. Any transaction related to our protocol is completely separate.

Reply:
hmm i see, back to the drawing board.